actiontext/lib/action_text/plain_text_conversion.rb

```diff
@@ -121,5 +122,33 @@ def break_if_nested_list(node, text)
           text
         end
       end
+
+      class BottomUpReducer
```

Comment:
```suggestion
      class BottomUpReducer # :nodoc:
```

https://intrip-actiontext-plaintext-o7gs.rails-docs-preview.pages.dev/api/classes/ActionText/PlainTextConversion/BottomUpReducer